wcfsetup/install/files/lib/data/user/avatar/UserAvatar.class.php

```diff
@@ -52,20 +54,29 @@ class UserAvatar extends DatabaseObject implements IUserAvatar {
 	 * Returns the physical location of this avatar.
 	 * 
 	 * @param	integer		$size
+	 * @param boolean $forceWebP
 	 * @return	string
 	 */
-	public function getLocation($size = null) {
-		return WCF_DIR . 'images/avatars/' . $this->getFilename($size);
+	public function getLocation($size = null, $forceWebP = null) {
+		return WCF_DIR . 'images/avatars/' . $this->getFilename($size, $forceWebP);
 	}
 	
 	/**
 	 * Returns the file name of this avatar.
 	 * 
 	 * @param	integer		$size
+	 * @param boolean $forceWebP
```

Comment:
This should be `?bool`. In fact it should be possible to actually use a scalar type hint here, because the parameter is new. Applies likewise above.